qubes-rpc/post-install.d/10-qubes-core-agent-features.sh

```diff
@@ -16,19 +15,18 @@ if [ -r /etc/os-release ]; then
     fi
     qvm-features-request os-distribution="$distro"
 
-    version=$(grep ^VERSION_ID= /etc/os-release)
-    version=${version#VERSION_ID=}
-    version=${version#\"}
-    version=${version%\"}
+    distro_like=$(awk -F '=' '/^ID_LIKE=/{ print $2 }' /etc/os-release)
+    qvm-features-request os-distribution-like="${distro_like}"
+
+    version=$(awk -F '[="]' '/^VERSION_ID=/{ print $3 }' /etc/os-release)
```

Comment:
This needs a bit more care - VERSION_ID value sometimes it quoted but sometimes not

Reply:
Last commits deals with this now.